packages/paste-website/src/component-examples/SkeletonLoaderExamples.ts

```diff
@@ -0,0 +1,344 @@
+          </Paragraph>
+        </Box>
+      ) : (
+        <Box width="500px">
+          <Stack orientation="vertical" spacing="space20">
+            <SkeletonLoader />
+            <SkeletonLoader />
+            <SkeletonLoader />
+            <SkeletonLoader />
+            <SkeletonLoader width="80px" />
+          </Stack>
+        </Box>
+      )}
+      <Box marginTop="space60">
+        <Stack orientation="horizontal" spacing="space60">
+          <Button variant="secondary" size="small" onClick={() => setLoaded(true)}>
+            Load content
+          </Button>
+          <Button variant="secondary" size="small" onClick={() => setLoaded(false)}>
+            Reset
+          </Button>
+        </Stack>
+      </Box>
+    </>
+  );
+};
+
+render(
+  <SkeletonLoaderParagraph />
+)
+`.trim();
+
+export const HeadingExample = `
+const SkeletonLoaderHeading = () => {
+  const [loaded, setLoaded] = React.useState(false);
+
+  return (
+    <>
+      {loaded ? (
+        <Stack orientation="vertical" spacing="space60">
+          <Heading as="h1" variant="heading10">
+            Heading one
+          </Heading>
+          <Heading as="h2" variant="heading20">
+            Heading two
+          </Heading>
+          <Heading as="h3" variant="heading30">
+            Heading three
+          </Heading>
+          <Heading as="h4" variant="heading40">
+            Heading four
+          </Heading>
+          <Heading as="h5" variant="heading50">
+            Heading five
+          </Heading>
+          <Heading as="h6" variant="heading60">
+            Heading six
+          </Heading>
+        </Stack>
+      ) : (
+        <Stack orientation="vertical" spacing="space60">
+          <SkeletonLoader height="44px" width="100px" />
+          <SkeletonLoader height="32px" width="100px" />
+          <SkeletonLoader height="28px" width="100px" />
+          <SkeletonLoader height="24px" width="100px" />
+          <SkeletonLoader height="20px" width="100px" />
+          <SkeletonLoader height="20px" width="100px" />
+        </Stack>
+      )}
+      <Box marginTop="space60">
+        <Stack orientation="horizontal" spacing="space60">
+          <Button variant="secondary" size="small" onClick={() => setLoaded(true)}>
+            Load content
+          </Button>
+          <Button variant="secondary" size="small" onClick={() => setLoaded(false)}>
+            Reset
+          </Button>
+        </Stack>
+      </Box>
+    </>
+  );
+};
+
+render(
+  <SkeletonLoaderHeading />
+)
+`.trim();
+
+export const ButtonsExample = `
+const SkeletonLoaderButtons = () => {
+  const [loaded, setLoaded] = React.useState(false);
+
+  return (
+    <>
+      {loaded ? (
+        <Stack orientation="horizontal" spacing="space40">
```

Comment:
Is there a benefit to including the layout component in the ternary render? Alternatively we could move the `Stack` outside of the ternary since for the both conditions we are rendering the same container.

Reply:
Good catch. Not on this on. Just got a little too free with the copy/paste here.

Reply:
Oh wait. I actually do need the stack there. If its not, I have to wrap the Buttons and Skeletons in a fragment, which then counts as the only child, so I would lose the margins.